conf/messages.en

```diff
@@ -419,6 +421,7 @@ messages__error__paye_invalid = Enter an employer PAYE number using only numbers
 messages__error__has_paye_establisher = Select whether the partnership has an employer PAYE number
 
 messages__establisher_individual_address__title = Establisher’s address
+messages__establisher_individual_confirmAddress__heading = Confirm {0}’s address
```

Comment:
This message is duplicate of the one on line 54, can we reuse that?

Reply:
added condition for H&S. please check